### PR TITLE
backport 4.3: remove opensuse microos

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/20162
https://github.com/uyuni-project/uyuni-docs/pull/2088

This supersedes https://github.com/uyuni-project/uyuni-docs/pull/2123

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Use grep. I am quite  sure you need to remove more things from other pages and files.

**Comment by keichwa** [comment]
I grep'ed for "microos" (= the macro) and found nothing more.  But I'll now also try "micro os".

**Comment by juliogonzalez** [comment]
> I grep'ed for "microos" (= the macro) and found nothing more. But I'll now also try "micro os".

Well, a quick grep filtering out translations and slemicro stuff, shows a lot of stuff:

```
$ grep -ir 'microos' *|grep -v l10n|grep -v 'suse-microos'
branding/pdf/entities.adoc::microos: MicroOS
modules/specialized-guides/pages/salt/salt-yomi.adoc:It also contains documentation about the different sections of the pillar, and some examples about how to parameterize installations based on openSUSE, MicroOS, or SLE.
modules/specialized-guides/pages/salt/salt-yomi.adoc:You can find examples for MicroOS or SLES in the example directory [path]``/usr/share/yomi/pillar/``.
modules/administration/pages/actions.adoc:For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-microos.adoc[].
modules/administration/pages/actions.adoc:For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-microos.adoc[].
modules/client-configuration/nav-client-configuration-guide.adoc:**** xref:clients-microos.adoc[openSUSE MicroOS]
modules/client-configuration/pages/clients-microos.adoc:= Registering {opensuse}{nbsp}{microos} Clients
modules/client-configuration/pages/clients-microos.adoc:This section contains information about registering clients running these {opensuse}{nbsp}{microos} operating systems:
modules/client-configuration/pages/clients-microos.adoc:* {opensuse}{nbsp}{microos}
modules/client-configuration/pages/clients-microos.adoc:The {microos} uses transactional updates.
modules/client-configuration/pages/clients-microos.adoc:Before you register {opensuse}{nbsp}{microos} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
modules/client-configuration/pages/clients-microos.adoc:.openSUSE MicroOS Channels - CLI
modules/client-configuration/pages/clients-microos.adoc:| {opensuse}{nbsp}{microos}
modules/client-configuration/pages/clients-microos.adoc:{opensuse}{nbsp}{microos} channels can be very large.
modules/client-configuration/pages/clients-microos.adoc:{opensuse}{nbsp}{microos} is not yet fully enabled, so there are some manual steps to trust the {productname} SSL certificate on {microos} clients.
modules/client-configuration/pages/clients-microos.adoc:// Not tested. From https://etherpad.nue.suse.com/p/uyuni-microos. --LKB 2021-04-15
modules/client-configuration/pages/clients-microos.adoc:To register your {opensuse}{nbsp}{microos} clients, you need a bootstrap repository.
modules/client-configuration/pages/supported-features-sle-micro.adoc:= Supported {micro} and {opensuse}{nbsp}{microos} Client Features
modules/client-configuration/pages/supported-features-sle-micro.adoc:{opensuse}{nbsp}{microos} is supported by the {suse} community.
modules/client-configuration/pages/supported-features-sle-micro.adoc:.Supported Features on {micro} and {opensuse}{nbsp}{microos} Operating Systems
modules/client-configuration/pages/supported-features-sle-micro.adoc:| {opensuse}{nbsp}{microos}
parameters.yml:  - attribute: microos
parameters.yml:    value: MicroOS
site.yml:    microos: MicroOS
```

**Review by jcayouette — CHANGES_REQUESTED** [review]
I will review after Julio's suggestions have been tackled.

**Comment by keichwa** [comment]
The important one it `modules/administration/pages/actions.adoc`, which is covered by this very PR ;)  The others are either ifdef'ed as uyuni content since quite some time or ... let me check... ;)

Anyway, I'll remove all these traces.

**Comment by juliogonzalez** [comment]
> The important one it `modules/administration/pages/actions.adoc`, which is covered by this very PR ;) The others are either ifdef'ed as uyuni content since quite some time or ... let me check... ;)
> 
> Anyway, I'll remove all these traces.

OK, but we were talking about grep :-)

And anyway, removing unneeded stuff is always good, as in the end sometimes antora still goes nuts and index things that should not be indexed :-D

File: modules/client-configuration/pages/supported-features-sle-micro.adoc [hunk 1]
```diff
@@ -19,181 +18,138 @@ The icons in this table indicate:
...
 | Organization custom states
 | {salt}
-| {salt}
 
 | System set manager (SSM)
 | {salt}
-| {salt}
 
 | Product migration
 | {cross}
```
Comment (keichwa) [inline modules/client-configuration/pages/supported-features-sle-micro.adoc:62]:
```suggestion
| {salt}
```

**Review by 0rnela — APPROVED** [review]

**Comment by keichwa** [comment]
I try to fix reported errors.  @juliogonzalez can you please check again?

**Comment by juliogonzalez** [comment]
> [suse-manager.tar.gz](https://github.com/uyuni-project/uyuni-docs/files/11220284/suse-manager.tar.gz) @juliogonzalez here is the tar. I hope I got it right.

Looks completely broken to me. Please open it and inspect it.

**Comment by keichwa** [comment]
> > Looks completely broken to me. Please open it and inspect it.
> 
> Yes, I tried to tar a subset, which was a bad idea (w/o js files...). Here now is the complete antora output:
> 
> [suse-manager.tar.gz](https://github.com/uyuni-project/uyuni-docs/files/11221706/suse-manager.tar.gz)

@juliogonzalez is it ok now?

**Comment by juliogonzalez** [comment]
> @juliogonzalez is it ok now?

I still see it broken. Did you check before pinging me? :-D

**Comment by keichwa** [comment]
> > @juliogonzalez is it ok now?
> 
> I still see it broken. Did you check before pinging me? :-D

I missed the `-` directory outside of the text output ;-(  And now it is too big for uploading.  I'll copy it to w3...

**Comment by juliogonzalez** [comment]
> > > @juliogonzalez is it ok now?
> > 
> > 
> > I still see it broken. Did you check before pinging me? :-D
> 
> I missed the `-` directory outside of the text output ;-( And now it is too big for uploading. I'll copy it to w3...

Please, do not copy the tarball.

Just upload the files, so the doc can be directly reviewed at w3.

**Review by juliogonzalez — APPROVED** [review]
- `grep -ir 'micro' *|grep -v branding/supplemental-ui|grep -v 'l10n'|more` looks OK
- Render looks OK (but be aware the the search function is broken for some reason, so makes the review hard).

However, it seems you left all suggestions from a previous review unattended here and you just fixed things on your branch. Looks Ok, but that's dangerous as you can miss things and makes reviewer's life harder to review if what was requested was done or not.

Worst case, reject all those suggestions telling that you fixed the problem on the branch directly.

In any case, seems things are looking good now. Not sure if you need to review the failing tests.

File: modules/administration/pages/actions.adoc [hunk 2]
```diff
@@ -71,10 +71,6 @@ To ensure that actions occur in the correct order, use action chains.
 For transactional update systems, an action chain is executed inside a single snapshot, until there is a reboot action.
 This can cause some limitations.
 
-ifeval::[{uyuni-content} == true]
-For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-microos.adoc[].
-endif::[]
-
 ifeval::[{suma-content} == true]
```
[the remark this answers is not in the document]
Comment (keichwa) [reply modules/administration/pages/actions.adoc]: Ok, done (externally).

**Comment by keichwa** [comment]
> * `grep -ir 'micro' *|grep -v branding/supplemental-ui|grep -v 'l10n'|more` looks OK
> 
>     * Render looks OK (but be aware the the search function is broken for some reason, so makes the review hard).
> 
> 
> However, it seems you left all suggestions from a previous review unattended here and you just fixed things on your branch. Looks Ok, but that's dangerous as you can miss things and makes reviewer's life harder to review if what was requested was done or not.
> 
> Worst case, reject all those suggestions telling that you fixed the problem on the branch directly.
> 
> In any case, seems things are looking good now. Not sure if you need to review the failing tests.

As the initial Description indicates, this PR is meant as a backport of https://github.com/uyuni-project/uyuni-docs/pull/2088 and shall supersede https://github.com/uyuni-project/uyuni-docs/pull/2123.  I tried to use all feedback I received.  Now I'll compare it with the main/master branch again, and create an additional PR for `master` if needed.

**Comment by keichwa** [comment]
> In any case, seems things are looking good now. Not sure if you need to review the failing tests.

Failing tests are not related to this PR.  I'll check the ifeval issue separately.

**Pull request merged**